app/editor/src/components/formik/form/FormikForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { Formik, FormikConfig, FormikValues } from 'formik';
+import React from 'react';
+
+export type FormikFormProps<
+  Values extends FormikValues = FormikValues,
+  ExtraProps = {},
+> = FormikConfig<Values> & ExtraProps;
+
+export interface IFormikFormProps<Values> extends FormikFormProps<Values> {}
+
+export const FormikForm = <Values extends FormikValues = FormikValues>({
```

Comment:
Base formik form so we don't have to have as much boilerplate code on formik forms.  We can add global features here.

Reply:
I like this